controllers/front/token.php

```diff
@@ -63,13 +63,13 @@ public function postProcess()
             ) {
                 $psCheckoutCart->paypal_token = $this->getToken();
                 $psCheckoutCart->paypal_token_expire = (new DateTime())->modify('+3550 seconds')->format('Y-m-d H:i:s');
-                $psCheckoutCart->update();
+                $psCheckoutCartRepository->save($psCheckoutCart);
             } else {
                 $psCheckoutCart = new PsCheckoutCart();
                 $psCheckoutCart->id_cart = (int) $this->context->cart->id;
                 $psCheckoutCart->paypal_token = $this->getToken();
                 $psCheckoutCart->paypal_token_expire = (new DateTime())->modify('+3550 seconds')->format('Y-m-d H:i:s');
-                $psCheckoutCart->add();
+                $psCheckoutCartRepository->save($psCheckoutCart);
             }
 
```

Comment:
duplicate line here : in all case we do this :
$psCheckoutCart->paypal_token = $this->getToken();
$psCheckoutCart->paypal_token_expire = (new DateTime())->modify('+3550 seconds')->format('Y-m-d H:i:s');
$psCheckoutCartRepository->save($psCheckoutCart);